src/server/console/CMakeLists.txt

```diff
@@ -9,7 +9,13 @@ add_library(
   logind-seat.c                             logind-seat.h
   logind-session.c                          logind-session.h
```

Comment:
These should go away, then? Are they even used? If we treat them the same as wayland protocols, we shouldn't have those checked in at all, but instead rely on the generated ones.